src/commonMain/kotlin/com/github/ajalt/colormath/Color.kt

```diff
@@ -56,6 +56,9 @@ interface Color {
     /** Convert this color to CIE LAB */
     fun toLAB(): LAB = toRGB().toLAB()
 
+    /** Convert this color to CIE LUV */
+    fun toLUV(): LUV = toRGB().toLUV()
```

Comment:
`RGB.toLUV()` isn't defined, so this is an infinite recursion. You'll need to write that function and add tests for it to `RGBTest.kt`.

Reply:
Oh, nice catch! My bad 😅